extensions/amp-analytics/0.1/vendors.js

```diff
@@ -107,6 +107,44 @@ export const ANALYTICS_CONFIG = {
       'xhrpost': false,
       'image': true
     }
+  },
+
+  'chartbeat': {
+    'vars': {
+      'decayTime': 15
+    },
+    'requests': {
+      'host': 'https://ping.chartbeat.net',
+      'interval': '${host}/ping?h=${domain}&' +
+        'p=${canonicalPath}&' +
+        'u=${clientId}&' +
```

Comment:
clientId var expects a scope parameter. If you guys have a cookie that you use on publisher domains, having that cookie name would make AMP read that cookie if available. If you don't want to share cookies and want to just use some other namespace, please specify that namespace.

Example: `${clientId(chartbeat-cookie-name)}`